### ISSUE TITLE
Downloads

### ISSUE THREAD

**Issue by norsborg** [VladThodo/behe-explorer#35]
>Thank you for your feedback!Please add below details so that we can help you a bit faster.

---

### Agreements
>Please check the below checkboxes to make sure that you are not spamming

---

- [ x] <b>I have read and accepted the [guidelines](https://github.com/VladThodo/behe-explorer/blob/development/CONTRIBUTING.md) for contributing to this project.</b>
- [ x] <b>I have checked the [open issues](https://github.com/VladThodo/behe-explorer/issues) and made shure that this is not a duplicate.</b>

---

### Short description
>Please add a short dscription of the issue.The description should contain:

---

 * BeHe ExloreR version:2.5.1
 * Android version that you are using:7.1.2

---

### Reproduction
>Please tell us how to reproduce this issue.

---
Go on https://download.lineageos.org/. Try to download a rom. The app will crash.

### Screenshots
>If you have any screenshots that can help please add them here 

---

**Comment by VladThodo** [comment]
Thanks for getting in touch! :)
Are you using any custom ROM? I'm trying to reproduce this issue on stock 6.0.1 and the downloads work fine.  I'll try to fix this, but I have to understand what is causing the app to crash first.

**Comment by norsborg** [comment]
Yes, I use LineageOS 14.1 for Galaxy S4 and unofficial CyanogenMod 13.0 for
Huawei P8. It doesn't work on both devices.

Am 04.05.2017 08:14 schrieb "Vlad Todosin" <notifications@github.com>:

**Comment by VladThodo** [comment]
BeHe uses the default android download manager to handle downloads. It seems that the download manager is different on each device, causing it to work or not. I'm working on an in-app download manager, so that BeHe will not rely on the android download manager anymore. Thanks for your report :)
Also, make sure that you have the new version of BeHe installed to get the updates: https://play.google.com/store/apps/details?id=com.vlath.beheexplorerfree
